subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -442,73 +442,67 @@ static void wifi_mgmt_event_handler(struct net_mgmt_event_callback *cb,
 	}
 }
 
-static int __wifi_args_to_params(size_t argc, char *argv[],
+static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv[],
 				 struct wifi_connect_req_params *params,
 				 enum wifi_iface_mode iface_mode)
 {
 	char *endptr;
 	int idx = 1;
-	const struct shell *sh = context.sh;
+	struct getopt_state *state;
+	int opt;
+	bool secure_connection = false;
+	static struct option long_options[] = {{"ssid", required_argument, 0, 's'},
+					       {"passphrase", required_argument, 0, 'p'},
+					       {"key-mgmt", required_argument, 0, 'k'},
+					       {"ieee-80211w", required_argument, 0, 'w'},
+					       {"bssid", required_argument, 0, 'm'},
+					       {"band", required_argument, 0, 'b'},
+					       {"channel", required_argument, 0, 'c'},
+					       {"help", no_argument, 0, 'h'},
+					       {0, 0, 0, 0}};
+	int opt_index = 0;
+	uint8_t band;
+	long channel = strtol(argv[idx], &endptr, 10);
+	const uint8_t all_bands[] = {WIFI_FREQ_BAND_2_4_GHZ,
+					WIFI_FREQ_BAND_5_GHZ,
+					WIFI_FREQ_BAND_6_GHZ};
```

Comment:
Could you modify this a bit to something like this
```
const uint8_t all_bands[] = {
	WIFI_FREQ_BAND_2_4_GHZ,
	WIFI_FREQ_BAND_5_GHZ,
	WIFI_FREQ_BAND_6_GHZ
};
```

Reply:
Done